RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectrons_cfi.py

```diff
@@ -0,0 +1,62 @@
+import FWCore.ParameterSet.Config as cms
+from RecoEcal.EgammaClusterProducers.hybridSuperClusters_cfi import *
+from RecoEcal.EgammaClusterProducers.multi5x5BasicClusters_cfi import *
+
+from RecoEgamma.EgammaIsolationAlgos.electronTrackIsolations_cfi import trkIsol03CfgV1,trkIsol04CfgV1
+
+lowPtGsfElectrons = cms.EDProducer(
+    "LowPtGsfElectronProducer",
+    
+    # input collections
+    previousGsfElectronsTag = cms.InputTag(""),
+    pflowGsfElectronsTag = cms.InputTag(""),
+    gsfElectronCoresTag = cms.InputTag("lowPtGsfElectronCores"),
+    pfMvaTag = cms.InputTag(""),
+    
+    # steering
+    applyPreselection = cms.bool(False), #@@ True
+    ecalDrivenEcalEnergyFromClassBasedParameterization = cms.bool(True), #@@ False
+    ecalDrivenEcalErrorFromClassBasedParameterization = cms.bool(True), #@@ False
+    applyAmbResolution = cms.bool(False),
+    useEcalRegression = cms.bool(False), #@@ True
+    useCombinationRegression = cms.bool(False), #@@ True
+    
+    # preselection parameters (ecal driven electrons)
+    preselection = cms.PSet(
+        minSCEtBarrel = cms.double(4.0),
+        minSCEtEndcaps = cms.double(4.0),
+        maxDeltaEtaBarrel = cms.double(0.02),
+        maxDeltaEtaEndcaps = cms.double(0.02),
+        maxDeltaPhiBarrel = cms.double(0.15),
+        maxDeltaPhiEndcaps = cms.double(0.15),
+        maxHOverEBarrel = cms.double(0.15),
```

Comment:
The name of these parameters have been modified in the master by #25699, see e.g. https://cmssdt.cern.ch/lxr/source/RecoEgamma/EgammaElectronProducers/plugins/GsfElectronBaseProducer.cc#0106

That PR is in the IBs since CMSSW_10_5_X_2019-01-24-1100: you must rename
"maxHOverEBarrel" -> "maxHOverEBarrelCone" + "maxHOverEBarrelTower" (and the same for the Endcaps)

Then, to remain able to test your fixes, you should rebase on a IB newer or equal than CMSSW_10_5_X_2019-01-24-1100

This update cannot be applied to the backport to 10_2_X

Reply:
@perrotta Do I force push? Or create a new PR?

Reply:
force push should be safe enough here...